chart/helm-operator/templates/deployment.yaml

```diff
@@ -145,6 +145,7 @@ spec:
         {{- end }}
         - --git-timeout={{ .Values.git.timeout }}
         - --git-poll-interval={{ .Values.git.pollInterval }}
+        - --git-default-chart-ref={{ .Values.git.defaultChartRef }}
```

Comment:
Wrap this in an `if` to maintain backwards compatibility:

```
{{- if .Values.git.defaultChartRef }}
- --git-default-chart-ref={{ .Values.git.defaultChartRef }}
{{- end }}
```

Reply:
@stefanprodan sure. Would you prefer the entire PR in one commit, or are you fine with me adding the review changes on top of the original commit?

Reply:
One commit is fine, amend your commit and force push it. Thanks